ozaria/site/views/play/level/tome/Spell.coffee

```diff
@@ -60,6 +61,8 @@ module.exports = class Spell
   setLanguage: (@language) ->
     @language = 'html' if @level.isType('web-dev')
     @displayCodeLanguage = utils.capitalLanguages[@language]
+    if @language in ['cpp', 'java', 'lua', 'coffeescript', 'python'] and not @languages[@language]
```

Comment:
why add all languages? Or we need java, lua too?

Reply:
i just copy from coco. maybe we can enable other langs in ozaria later too ? i think it is not blocking anyway